tests/test_config_reader.py

```diff
@@ -20,3 +20,28 @@ def test_parse_config_files():
         assert pytest_exit_object.type == SystemExit 
         assert pytest_exit_object.value.code == 42  
         assert pytest_exit_object.sys.stdout.getline().strip() == "No file named config.yml found in config directory {}. Exiting...".format(THIS_DIR)
+    
+@pytest.mark.parametrize(
+    "input,expected_name,expected_path",
+    [
+        ({"general_parameters": {"data_output_filename": 1.1, "data_output_path": 2.2}}, "1.1", "2.2"),
+        ({"general_parameters": {"data_output_filename": "my_data", "data_output_path": 2.2}}, "my_data", "2.2"),
+        ({"general_parameters": {"data_output_filename": 1.1, "data_output_path": "desktop"}}, "1.1", "Desktop")
+    ],
+)
+def test_get_out_params(input, expected_name, expected_path, tmpdir):
+    out_name, out_path = get_output_params(input)
+    split_out_name = out_name.split("_")
```

Comment:
I don't quite get why you are splitting the output name.  What are you trying to test here?  This seems to be overly complicated, but I am not sure what your goal is so I may be wrong.

Reply:
Here I'm just wanting to see if it appended the correct name to the string based on the input since the function should append the input name with a timestamp. Appended part is split by the underscore. I guess I could check the full name with the timestamp, but is there a way to know the exact time that the function was called, so I can know what the appropriate timestamp should be?

I know how to check on pycharm, but I also like putting on here for the version history just in case I want to do a variation of a past attempt.  But maybe this isn't the most appropriate way to do that?